legacy/default.json

```
@@ -682,7 +682,7 @@
   "AJUDA_PleaseDonate": "Para eu conseguir ficar online 24/7 (bem que as vezes eu fico offline para ir tomar água, ir almoçar no Outback e as vezes tomar um frappuccino sem café no Starbucks) eu preciso pagar uma hospedagem mensal!\n\nAtualmente eu estou hospedada em um dedicado de 32GBs/4.00GHz na SoYouStart! Isso custa mais ou menos uns 260R$ por mês!\n\nPor isto, se você tiver alguma grana de sobrando porque você resolveu não comprar aquele pão de queijo que parecia estar meio mofado lá no Barzinho do Vieirinha, que tal doar para mim? Eu estou com fome... <:loritta_comendo_pudim:378551064313724928> https://loritta.website/donate",
   "AJUDA_DonationTitle": "Quer me ajudar?",
   "CONGAPARROT_Description": "Faça uma linha de conga!",
-  "CONGAPARROT_InvalidRange": "Infelizmente a conga precisa ser entre 1 e 50, tente outro número...",
+  "CONGAPARROT_InvalidRange": "Infelizmente a conga precisa ser entre 1 e 30, tente outro número...",
```

I think using placeholders would be better, like:
```suggestion
  "CONGAPARROT_InvalidRange": "Infelizmente a conga precisa ser entre 1 e {0}, tente outro número...",
```